### ISSUE TITLE
Boot warning

### ISSUE THREAD

**Issue by moul** [scaleway/image-archlinux#14]
```
[ TIME ] Timed out waiting for device dev-ttyS0.device.
[DEPEND] Dependency failed for Serial Getty on ttyS0.
[  OK  ] Reached target Login Prompts.
[  OK  ] Reached target Multi-User System.
[  OK  ] Reached target Graphical Interface.
```

**Comment by inigoserna** [comment]
This is due the same cause than issue #14, you can mark it as duplicated.

**Comment by moul** [comment]
This is fixed using 226376e15eb6d138587fd4a6389debf107f18b37, the next build of the image will use a more recent kernel

For now you can change your bootscript manually: https://www.scaleway.com/docs/bootscript-and-how-to-use-it/
